frontend/src/reports/options/Options.svelte

```diff
@@ -2,11 +2,17 @@
   import { urlFor } from "../../helpers";
   import { _ } from "../../i18n";
   import OptionsTable from "./OptionsTable.svelte";
+  import ThemeToggle from "./ThemeToggle.svelte";
 
   export let fava_options: Record<string, string>;
   export let beancount_options: Record<string, string>;
 </script>
 
+<h2>
+  {_("Theme style")}
+</h2>
+<ThemeToggle />
```

Comment:
Can you reuse the ModeSwitch.svelte component instead of creating a new component for this?